build

```diff
@@ -2,6 +2,31 @@
 
 set -e
 
+# Cross compatibility with osx
+# http://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac
```

Comment:
Just noticed a simpler version used by kubernetes here: https://github.com/kubernetes/kubernetes/blob/master/hack/lib/init.sh#L99
What do you think?